doc/source/bibtex/ref.bib

```diff
@@ -178,6 +178,30 @@ @article{Toby:15
 pages = {15--21}
 }
 
+@inproceedings{Vogelgesang:16,
+author = {Vogelgesang, Matthias and Rota, Lorenzo and Ardila Perez, Luis E. and Caselle, Michele and Chilingaryan, Suren and Kopmann, Andreas},
+title = { High-throughput data acquisition and processing for real-time x-ray imaging },
+journal = {Proc. SPIE},
+volume = {9967},
+pages = {996715-996715-9},
+year = {2016},
+doi = {10.1117/12.2237611},
+}
+
+@inproceedings{vogelgesang2012ufo,
+author = {Vogelgesang, M. and Chilingaryan, S. and Rolo, T.  dos Santos and Kopmann, A.},
+booktitle = {Proceedings of The 14th IEEE Conference on High Performance Computing and Communication \& The 9th IEEE International Conference on Embedded Software and Systems (HPCC-ICESS)},
+title = {UFO: A Scalable GPU-based Image Processing Framework for On-line Monitoring},
```

Comment:
You can rather have this in direct citations rather than references:
https://github.com/tomopy/tomopy/blob/master/doc/source/bibtex/cite.bib
and add:
...and also cite [A2] if you use ASTRA, and [A3] if you use UFO algorithms).
Please let me know before merge.